Libraries/Azure.psm1

```diff
@@ -2093,7 +2093,7 @@ Function Set-SRIOVinAzureVMs {
                 $TestVMData = $VMData | Where-Object {$_.ResourceGroupName -eq $ResourceGroup `
                     -and $_.RoleName -eq $VMName }
 
-                $isVmAlive = Is-VmAlive -AllVMDataObject $TestVMData
+                $isVmAlive = Is-VmAlive -AllVMDataObject $TestVMData -MaxRetryCount 50
```

Comment:
what's the situation that it needs 50 retries? It might wait a lot of time if the VM is not accessible

Reply:
This is one only in the function: Set-SRIOVinAzureVMs. It seems Azure networking takes a longer amount of time to come back after the SRIOV is disabled for 3 nics. The default retries are set to 20, which means around 1 minute as the sleep interval is 3 seconds. Bumping it to 50 is not a huge deal, as it can prolong the test case by 2 minutes tops. In my tests, the VM comes back after the sriov disable step after around 36-48 retries. I think I have to bump it to 60, to be on the safe side.

Reply:
@lizzha I have added a parameter for the isalive check, so that only the TC that is failing changes the retry count. All the other TC will not be affected.

Reply:
@lizzha will fix this issue on another PR, I have removed the code.